docs/solution-architecture/systemContext.md

```diff
@@ -99,6 +101,7 @@ Furthermore, ZAC integrates with the following external services:
 | [Mailjet](https://www.mailjet.com/)                                                  | Email service.                                                                     | Send emails to employees. Only used for sending e-mails, not for managing e-mail templates (this is done in ZAC itself).            | <ul><li>Mailjet Email API (latest version)</li></ul>                                            |
 | [SmartDocuments](https://www.smartdocuments.eu/)                                     | Document creation service. Maybe used to create documents in your own look & feel. | Start a document creation 'wizard' with pre-filled zaak data where the resulting document is stored by SmartDocuments in Open Zaak. | <ul><li>SmartDocuments REST API (latest version)</li></ul>                                      |
 | [VNG Referentielijsten](https://vng-realisatie.github.io/gemma-zaken/ontwikkelaars/) | Centralized reference data service.                                                | Retrieve reference data such as 'communication channels'.                                                                           | <ul><li>Referentielijsten & Selectielijst API 0.6.0 (see below)</li></ul>                       |
+| [OfficeConverter](https://github.com/EugenMayer/officeconverter)                     | Document conversion service.                                                       | Convert office documents (like .docx) to PDF for preview purposes.                                                                  | <ul><li>OfficeConverter REST API 1.5.0</li></ul>                                                |
```

Comment:
Office converter belongs in the list of components, as it is one that is hosted within K8S.

Reply:
Indeed. The office convertor is component that more or less belongs to ZAC. It is definitely not an external service.